cips/cip-network-upgrade.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Coordinated network upgrades
+description: Protocol for coordinating major network upgrades
+author: Callum Waters (@cmwaters)
+discussions-to: URL
```

Comment:
post a URL please from forum for discussions

Reply:
Done although it's a bit of a chicken and egg problem. I will update the forum discussion to link to the actual document instead of the PR for the document after merging this